bazel/spec-bundling/index.bzl

```diff
@@ -61,23 +65,34 @@ def spec_bundle(
         fail("The spec-bundling target %s is declared as browser test. In order to be able " +
              "to construct an AMD module name, the `workspace_name` attribute needs to be set.")
 
-    # Browser tests (Karma) need named AMD modules to load.
-    # TODO(devversion): consider updating `@bazel/concatjs` to support loading JS files directly.
-    esbuild_rule = esbuild_amd if is_browser_test else esbuild_esm_bundle
-    amd_name = "%s/%s/%s" % (workspace_name, package_name, name + "_spec") if is_browser_test else None
+    esbuild_rule = None
+
+    if is_browser_test:
+        esbuild_rule = esbuild_amd
+        amd_name = "%s/%s/%s" % (workspace_name, package_name, name + "_spec") if is_browser_test else None
```

Comment:
Is the `is_browser_test` check here necessary? This whole block is inside of a `if is_browser_test:`.

Reply:
good catch